src/01-Learning-JS/03-operators.md

```diff
@@ -216,7 +228,12 @@ First, go back to the last chapter and grab the suite of primitives that you cre
 5. Check your work, as you go, by opening the browser console with the **Inspect Element** tool.
 
 ```javascript
-// Write your code in here
+let FavFilm = 1
```

Comment:
You should be testing some operations and how they evaluate your suite of primitives that you copy and paste into this notebook from chapter 1.2. For example, here's your primitives from 1.2:

```js
let hello = "DND!"
let critSuccess = 20
let truth = true
let deceive = false
null
let x
console.log(x) //Output: undefined
let rollforInitiative = {
  great: "Yes!",
  bad: "No!",
}
rollforInitiative.excited = "LetsDoThis!"
```

Now you can write some logs with operators to evaluate

```js
console.log(hello != "Hello") // should evaluate as `false`
```